proto/babylon/btcstaking/v1/params.proto

```diff
@@ -55,6 +55,8 @@ message Params {
   ];
   // max_active_finality_providers is the maximum number of active finality providers in the BTC staking protocol
   uint32 max_active_finality_providers = 13;
+  // base gas fee for delegation creation in Satoshi
+  int64 delegation_creation_base_gas_fee_sat = 14;
```

Comment:
imo this should be an `uint64` as negative value do not make sense here and we also avoid type conversion in `ConsumeGas`

Reply:
I thought we should keep consistent with other parameters e.g., `min_slashing_tx_fee_sat` and `unbonding_fee_sat`

Reply:
ahh I see confusion!

This should be `delegation_creation_base_gas_fee` , not `delegation_creation_base_gas_fee_sat` i.e the additional gas fee is not in satoshis but in cosmos gas unit. 

`min_slashing_tx_fee_sat` and `unbonding_fee_sat` are used as `int64` becouse thats what `btcd` library uses, so in those cases minimize amount of type conversions/